install.sh

```diff
@@ -447,8 +447,8 @@ fi
 cd "/usr" || exit 1
 
 ####################################################################### script
-USABLE_GIT="$(command -v git)"
-if [[ -z "${USABLE_GIT}" ]]
+USABLE_GIT="$(command -V git | awk '{ print $3 }')"
```

Comment:
Why don't we just check if
```
command -v git
```
starts with `.`, and if so, replace the `.` with the current working directory?

Reply:
Note that relative path components in `PATH` don't have to start with a `.`: `PATH=mybin:$PATH command -v git` can return `mybin/git`.

Better to `readlink -f` the result instead, that always Does The Right Thing.

Reply:
Yea as mentioned in the pr this appending was a thought of mine. But checking the first character seemed like adding a lot of complexity. What would work is checking if it is a / but then there seems to be so much that could go wrong.

I also thought about ``readlink`` but is that a standard preinstalled tool on all systems?

Reply:
> I also thought about `readlink` but is that a standard preinstalled tool on all systems?

`readlink` is standard fare on macOS, going back at least to Mojave.

As for Linux, given that `readlink` is a part of GNU `coreutils`, I'm pretty sure any distro missing `readlink` is also missing enough other essential utilities that we actually _want_ the installer to fail, noisily.

Reply:
Could try `realpath` if it's only for Linux anyway?

Reply:
Alternatively: https://stackoverflow.com/a/31605674 (the `cd` thing) is the standard one

Reply:
> Could try `realpath` if it's only for Linux anyway?

The bug this pr tries to fix exists on mac and linux at the moment.

Reply:
> Could try `realpath` if it's only for Linux anyway?

Worth noting `/bin/realpath` is now a thing on Ventura. So in a few years it would be a viable solution for Homebrew on macOS too.